Build failure: radicle-cli-0.6.1

### Steps To Reproduce

Steps to reproduce the behavior:
1. Try to build `radicle-cli` on the `nixos-unstable` branch (fe2ecaf706a5907b5e54d979fbde4924d84b65fc).

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/hiprh120fdqp0c0py21i0y2ljyky9wy5-source
source root is source
Executing cargoSetupPostUnpackHook
Finished cargoSetupPostUnpackHook
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/cargo-vendor-dir/Cargo.lock
Finished cargoSetupPostPatchHook
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
@nix { "action": "setPhase", "phase": "buildPhase" }
building
Executing cargoBuildHook
++ env CC_x86_64-unknown-linux-gnu=/nix/store/nlgyw2fv0cm8rkz8qm1jyw78vyif1bl9-gcc-wrapper-12.2.0/bin/cc CXX_x86_64-unknown-linux-gnu=/nix/store/nlgyw2fv0cm8rkz8qm1jyw78vyif1bl9-gcc-wrapper-12.2.0/bin/c++ CC_x86_64-unknown-linux-gnu=/nix/store/nlgyw2fv0cm8rkz8qm1jyw78vyif1bl9-gcc-wrapper-12.2.0/bin/cc CXX_x86_64-unknown-linux-gnu=/nix/store/nlgyw2fv0cm8rkz8qm1jyw78vyif1bl9-gcc-wrapper-12.2.0/bin/c++ cargo build -j 16 --target x86_64-unknown-linux-gnu --frozen --release
   Compiling libc v0.2.125
   Compiling proc-macro2 v1.0.37
   Compiling syn v1.0.92
   Compiling unicode-xid v0.2.2
   Compiling serde_derive v1.0.137
   Compiling serde v1.0.137
   Compiling cfg-if v1.0.0
   Compiling autocfg v1.1.0
   Compiling version_check v0.9.4
   Compiling memchr v2.5.0
   Compiling typenum v1.15.0
   Compiling log v0.4.16
   Compiling pkg-config v0.3.25
   Compiling lazy_static v1.4.0
   Compiling futures-core v0.3.21
   Compiling once_cell v1.10.0
   Compiling pin-project-lite v0.2.9
   Compiling subtle v2.4.1
   Compiling byteorder v1.4.3
   Compiling futures-task v0.3.21
   Compiling futures-channel v0.3.21
   Compiling hex v0.4.3
   Compiling futures-util v0.3.21
   Compiling crossbeam-utils v0.8.8
   Compiling opaque-debug v0.3.0
   Compiling generic-array v0.14.5
   Compiling slab v0.4.6
   Compiling num-traits v0.2.14
   Compiling memoffset v0.6.5
   Compiling futures-io v0.3.21
   Compiling cpufeatures v0.2.2
   Compiling ppv-lite86 v0.2.16
   Compiling block-padding v0.2.1
   Compiling crossbeam-epoch v0.9.8
   Compiling serde_json v1.0.80
   Compiling tinyvec_macros v0.1.0
   Compiling itoa v1.0.1
   Compiling tinyvec v1.6.0
   Compiling ahash v0.7.6
   Compiling futures-sink v0.3.21
   Compiling lock_api v0.4.7
   Compiling rayon-core v1.9.2
   Compiling quote v1.0.18
   Compiling crossbeam-queue v0.3.5
   Compiling crc32fast v1.3.2
   Compiling pin-utils v0.1.0
   Compiling ryu v1.0.9
   Compiling rayon v1.5.2
   Compiling tracing-core v0.1.26
   Compiling matches v0.1.9
   Compiling bitflags v1.3.2
   Compiling indexmap v1.8.1
   Compiling getrandom v0.1.16
   Compiling scopeguard v1.1.0
   Compiling signal-hook v0.3.13
   Compiling spin v0.5.2
   Compiling untrusted v0.7.1
   Compiling regex-automata v0.1.10
   Compiling keccak v0.1.0
   Compiling async-trait v0.1.53
   Compiling percent-encoding v2.1.0
   Compiling arrayvec v0.7.2
   Compiling form_urlencoded v1.0.1
   Compiling crunchy v0.2.2
   Compiling unicode-normalization v0.1.19
   Compiling jobserver v0.1.24
   Compiling getrandom v0.2.6
   Compiling num_cpus v1.13.1
   Compiling signal-hook-registry v1.4.0
   Compiling rand_core v0.6.3
   Compiling cc v1.0.73
   Compiling socket2 v0.4.4
   Compiling mio v0.8.2
   Compiling rand_chacha v0.3.1
   Compiling bstr v0.2.17
   Compiling unicode-bidi v0.3.8
   Compiling radium v0.6.2
   Compiling rand v0.8.5
   Compiling cmake v0.1.48
   Compiling idna v0.2.3
   Compiling fastrand v1.7.0
   Compiling quick-error v2.0.1
   Compiling proc-macro-error-attr v1.0.4
   Compiling minimal-lexical v0.2.1
   Compiling parking_lot_core v0.8.5
   Compiling aho-corasick v0.7.18
   Compiling proc-macro-error v1.0.4
   Compiling regex-syntax v0.6.25
   Compiling sha2-asm v0.6.2
   Compiling ring v0.16.20
   Compiling libz-sys v1.1.6
   Compiling openssl-sys v0.9.72
   Compiling nom v7.1.1
   Compiling base64 v0.13.0
   Compiling smallvec v1.8.0
   Compiling minicbor v0.13.2
   Compiling adler v1.0.2
   Compiling tiny-keccak v2.0.2
   Compiling miniz_oxide v0.5.1
   Compiling crossbeam-deque v0.8.1
   Compiling crossbeam-channel v0.5.4
   Compiling base64ct v1.5.0
   Compiling anyhow v1.0.57
   Compiling parking_lot_core v0.9.3
   Compiling same-file v1.0.6
   Compiling httparse v1.7.1
   Compiling openssl v0.10.38
   Compiling walkdir v2.3.2
   Compiling regex v1.5.5
   Compiling libgit2-sys v0.12.26+1.3.0
   Compiling git-hash v0.8.0
   Compiling waker-fn v1.1.0
   Compiling fnv v1.0.7
   Compiling parking v2.0.0
   Compiling futures-lite v1.12.0
   Compiling synstructure v0.12.6
   Compiling btoi v0.4.2
   Compiling num-integer v0.1.45
   Compiling instant v0.1.12
   Compiling itoa v0.4.8
   Compiling crossbeam v0.8.1
   Compiling im v15.1.0
   Compiling camino v1.0.7
   Compiling remove_dir_all v0.5.3
   Compiling bytesize v1.1.0
   Compiling sha1_smol v1.0.0
   Compiling human_format v1.0.3
   Compiling rustls v0.20.4
   Compiling cache-padded v1.2.0
   Compiling semver v1.0.9
   Compiling concurrent-queue v1.2.2
   Compiling sha1 v0.6.1
   Compiling prodash v16.1.3
   Compiling tempfile v3.3.0
   Compiling rand_core v0.5.1
   Compiling git-actor v0.6.0
   Compiling git-validate v0.5.3
   Compiling constant_time_eq v0.1.5
   Compiling cfg-if v0.1.10
   Compiling const-oid v0.6.2
   Compiling arrayvec v0.5.2
   Compiling const-oid v0.7.1
   Compiling native-tls v0.2.10
   Compiling hashbrown v0.12.0
   Compiling arrayref v0.3.6
   Compiling der v0.5.1
   Compiling der v0.4.5
   Compiling git-object v0.15.1
   Compiling dashmap v5.3.2
   Compiling event-listener v2.5.2
   Compiling home v0.5.3
   Compiling wasm-bindgen-shared v0.2.80
   Compiling rustc-hex v2.1.0
   Compiling git-tempfile v1.0.6
   Compiling spki v0.5.4
   Compiling spki v0.4.1
   Compiling thiserror-impl v1.0.31
   Compiling zeroize_derive v1.3.2
   Compiling futures-macro v0.3.21
   Compiling tracing-attributes v0.1.21
   Compiling tokio-macros v1.7.0
   Compiling zeroize v1.4.3
   Compiling impl-trait-for-tuples v0.2.2
   Compiling pin-project-internal v1.0.10
   Compiling flate2 v1.0.23
   Compiling thiserror v1.0.31
   Compiling tracing v0.1.34
   Compiling minicbor-derive v0.8.0
   Compiling maybe-async v0.2.6
   Compiling blake2b_simd v0.5.11
   Compiling pin-project v1.0.10
   Compiling ff v0.10.1
   Compiling tap v1.0.1
   Compiling wyz v0.2.0
   Compiling funty v1.1.0
   Compiling winapi v0.3.9
   Compiling convert_case v0.4.0
   Compiling static_assertions v1.1.0
   Compiling group v0.10.0
   Compiling parking_lot v0.11.2
   Compiling bitvec v0.20.4
   Compiling pkcs8 v0.8.0
   Compiling pkcs8 v0.7.6
   Compiling ff v0.11.0
   Compiling dashmap v4.0.2
   Compiling polling v2.2.0
   Compiling filebuffer v0.4.0
   Compiling num-iter v0.1.43
   Compiling futures-executor v0.3.21
   Compiling data-encoding v2.3.2
   Compiling byte-slice-cast v1.2.1
   Compiling futures v0.3.21
   Compiling bumpalo v3.9.1
   Compiling wasm-bindgen-backend v0.2.80
   Compiling async-io v1.6.0
   Compiling group v0.11.0
   Compiling fixed-hash v0.7.0
   Compiling git-traverse v0.10.1
   Compiling git-diff v0.11.1
   Compiling git-ref-format-core v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling async-channel v1.6.1
   Compiling git-packetline v0.12.4
   Compiling uluru v3.0.0
   Compiling bitmaps v2.1.0
   Compiling generic-array v0.12.4
   Compiling git-ref-format-macro v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling byte-tools v0.3.1
   Compiling unicode-segmentation v1.9.0
   Compiling base16ct v0.1.1
   Compiling atomic-waker v1.0.0
   Compiling async-task v4.2.0
   Compiling foreign-types-shared v0.1.1
   Compiling clru v0.5.0
   Compiling encoding_rs v0.8.31
   Compiling blocking v1.2.0
   Compiling foreign-types v0.3.2
   Compiling heck v0.3.3
   Compiling block-padding v0.1.5
   Compiling wasm-bindgen-macro-support v0.2.80
   Compiling parking_lot v0.12.0
   Compiling lnk-cryptovec v0.6.1
   Compiling sized-chunks v0.6.5
   Compiling uint v0.9.3
   Compiling rlp-derive v0.1.0
   Compiling git-lock v1.0.1
   Compiling webpki v0.21.4
   Compiling sct v0.6.1
   Compiling rand_xoshiro v0.6.0
   Compiling num-bigint v0.2.6
   Compiling wasm-bindgen v0.2.80
   Compiling os_str_bytes v3.1.0
   Compiling ipnet v2.5.0
   Compiling wasm-bindgen-macro v0.2.80
   Compiling bytes v1.1.0
   Compiling either v1.6.1
   Compiling url v2.2.2
   Compiling toml v0.5.9
   Compiling tokio v1.18.0
   Compiling hashbrown v0.11.2
   Compiling digest v0.9.0
   Compiling block-buffer v0.9.0
   Compiling crypto-mac v0.11.1
   Compiling sha3 v0.9.1
   Compiling hmac v0.11.0
   Compiling cipher v0.3.0
   Compiling sha2 v0.9.9
   Compiling proc-macro-crate v1.1.3
   Compiling uuid v0.8.2
   Compiling salsa20 v0.8.1
   Compiling signature v1.3.2
   Compiling crypto-bigint v0.2.11
   Compiling crypto-bigint v0.3.2
   Compiling rlp v0.5.1
   Compiling git-url v0.3.5
   Compiling sec1 v0.2.1
   Compiling impl-rlp v0.3.0
   Compiling itertools v0.10.3
   Compiling impl-serde v0.3.2
   Compiling parity-scale-codec-derive v2.3.1
   Compiling elliptic-curve v0.10.6
   Compiling elliptic-curve v0.11.12
   Compiling ecdsa v0.12.4
   Compiling universal-hash v0.4.1
   Compiling ethbloom v0.11.1
   Compiling chacha20 v0.8.1
   Compiling poly1305 v0.7.2
   Compiling pbkdf2 v0.9.0
   Compiling sha-1 v0.9.8
   Compiling curve25519-dalek v3.2.0
   Compiling jwalk v0.6.0
   Compiling aead v0.4.3
   Compiling git-features v0.17.0
   Compiling data-encoding-macro-internal v0.1.10
   Compiling git-transport v0.13.1
   Compiling git-pack v0.14.0
   Compiling git-ref v0.9.1
   Compiling versions v3.0.3
   Compiling http v0.2.7
   Compiling git-protocol v0.12.1
   Compiling serde_bytes v0.11.6
   Compiling smol_str v0.1.23
   Compiling parity-scale-codec v2.3.1
   Compiling rustls v0.19.1
   Compiling impl-codec v0.5.1
   Compiling primitive-types v0.10.1
   Compiling git-odb v0.24.0
   Compiling strum_macros v0.21.1
   Compiling lnk-thrussh-encoding v0.1.0
   Compiling block-buffer v0.7.3
   Compiling digest v0.8.1
   Compiling async-process v1.4.0
   Compiling blake2s_simd v0.5.11
   Compiling webpki v0.22.0
   Compiling num-rational v0.2.4
   Compiling num-complex v0.2.4
   Compiling arc-swap v1.5.0
   Compiling rustc-hash v1.1.0
   Compiling radicle-std-ext v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling half v1.8.2
   Compiling fake-simd v0.1.2
   Compiling unsigned-varint v0.5.1
   Compiling opaque-debug v0.2.3
   Compiling strum v0.21.0
   Compiling serde_cbor v0.10.2
   Compiling ethereum-types v0.12.1
   Compiling git-ref-format v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling lnk-thrussh-agent v0.1.0
   Compiling tokio-util v0.7.1
   Compiling data-encoding-macro v0.1.12
   Compiling chacha20poly1305 v0.9.0
   Compiling cargo-platform v0.1.2
   Compiling sct v0.7.0
   Compiling iovec v0.1.4
   Compiling rpassword v4.0.5
   Compiling secstr v0.3.2
   Compiling try-lock v0.2.3
   Compiling base-x v0.2.10
   Compiling nonzero_ext v0.2.0
   Compiling multibase v0.9.1
   Compiling want v0.3.0
   Compiling multihash v0.11.4
   Compiling js-sys v0.3.57
   Compiling cargo_metadata v0.14.2
   Compiling h2 v0.3.13
   Compiling ethabi v15.0.0
   Compiling automerge-protocol v0.1.0 (https://github.com/automerge/automerge-rs.git?rev=e72571962b51c2f0726fb534890ef3b4f7c74dfc#e7257196)
   Compiling http-body v0.4.4
   Compiling nonempty v0.7.0
   Compiling net2 v0.2.37
   Compiling fxhash v0.2.1
   Compiling futures-timer v3.0.2
   Compiling openssl-probe v0.1.5
   Compiling tower-service v0.3.1
   Compiling bit-vec v0.6.3
   Compiling httpdate v1.0.2
   Compiling leb128 v0.2.5
   Compiling maplit v1.0.2
   Compiling dyn-clone v1.0.5
   Compiling bit-set v0.5.2
   Compiling web-sys v0.3.57
   Compiling mio v0.6.23
   Compiling radicle-data v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling num v0.2.1
   Compiling itertools v0.9.0
   Compiling link-canonical-derive v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling hyper v0.14.18
   Compiling libusb1-sys v0.6.2 (https://github.com/a1ien/rusb.git?rev=050f5091c7b0fb69c0fc25edec0910abe8afadf1#050f5091)
   Compiling dirs-sys v0.3.7
   Compiling atty v0.2.14
   Compiling hashbrown v0.8.2
   Compiling lazycell v1.3.0
   Compiling num_threads v0.1.5
   Compiling time-macros v0.2.4
   Compiling pin-project-internal v0.4.29
   Compiling mio-extras v2.0.6
   Compiling automerge-frontend v0.1.0 (https://github.com/automerge/automerge-rs.git?rev=e72571962b51c2f0726fb534890ef3b4f7c74dfc#e7257196)
   Compiling link-canonical v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling fraction v0.9.0
   Compiling tokio-native-tls v0.3.0
   Compiling fancy-regex v0.7.1
   Compiling tokio-rustls v0.23.3
   Compiling sha2 v0.8.2
   Compiling webpki-roots v0.22.3
   Compiling crypto-mac v0.8.0
   Compiling time v0.3.9
   Compiling oid v0.2.1
   Compiling git-trailers v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling iso8601 v0.4.1
   Compiling xorf v0.7.2
   Compiling sized-vec v0.3.0
   Compiling num-cmp v0.1.0
   Compiling fixedbitset v0.2.0
   Compiling base58 v0.1.0
   Compiling bytecount v0.6.2
   Compiling ahash v0.3.8
   Compiling jsonschema v0.13.3
   Compiling base58check v0.1.0
   Compiling petgraph v0.5.1
   Compiling hyper-tls v0.5.0
   Compiling hyper-rustls v0.23.0
   Compiling picky-asn1 v0.3.3
   Compiling blake2 v0.9.2
   Compiling rand_pcg v0.2.1
   Compiling rand_chacha v0.2.2
   Compiling lru v0.7.5
   Compiling ripemd160 v0.9.1
   Compiling serde_urlencoded v0.7.1
   Compiling password-hash v0.2.3
   Compiling rustls-pemfile v0.3.0
   Compiling inotify-sys v0.1.5
   Compiling siphasher v0.3.10
   Compiling base64 v0.12.3
   Compiling bech32 v0.7.3
   Compiling rusb v0.9.1
   Compiling mime v0.3.16
   Compiling reqwest v0.11.10
   Compiling phf_shared v0.8.0
   Compiling inotify v0.7.1
   Compiling pin-project v0.4.29
   Compiling rand v0.7.3
   Compiling picky-asn1-der v0.2.5
   Compiling k256 v0.9.6
   Compiling no-std-compat v0.4.1
   Compiling quinn-proto v0.7.3
   Compiling bincode v1.3.3
   Compiling async-stream-impl v0.3.3
   Compiling hidapi-rusb v1.3.1
   Compiling quanta v0.4.1
   Compiling filetime v0.2.16
   Compiling mio v0.7.14
   Compiling socket2 v0.3.19
   Compiling nix v0.13.1
   Compiling bs58 v0.4.0
   Compiling bytes v0.5.6
   Compiling async-stream v0.3.3
   Compiling notify v4.0.17
   Compiling futures_codec v0.4.1
   Compiling governor v0.3.2
   Compiling ethers-core v0.6.3
   Compiling phf_generator v0.8.0
   Compiling picky-asn1-x509 v0.6.1
   Compiling directories v3.0.2
   Compiling link-async v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling quinn v0.7.2
   Compiling deadpool v0.7.0
   Compiling if-watch v0.2.2
   Compiling Inflector v0.11.4
   Compiling async-lock v2.5.0
   Compiling backoff v0.3.0
   Compiling globset v0.4.8
   Compiling bloom-filters v0.1.2
   Compiling rand_pcg v0.3.1
   Compiling num-rational v0.3.2
   Compiling void v1.0.2
   Compiling nonzero_ext v0.3.0
   Compiling radium v0.3.0
   Compiling phf_codegen v0.8.0
   Compiling aes v0.7.5
   Compiling bitvec v0.17.4
   Compiling ctr v0.7.0
   Compiling tracing-futures v0.2.5
   Compiling auto_impl v0.5.0
   Compiling ethers-contract-abigen v0.6.3
   Compiling nom v5.1.2
   Compiling heck v0.4.0
   Compiling termcolor v1.1.3
   Compiling os_str_bytes v6.0.0
   Compiling clap_lex v0.2.0
   Compiling clap_derive v3.1.7
   Compiling terminfo v0.7.3
   Compiling rfc6979 v0.1.0
   Compiling serde-aux v3.0.1
   Compiling crypto-common v0.1.3
   Compiling block-buffer v0.10.2
   Compiling nix v0.23.1
   Compiling bytemuck v1.9.1
   Compiling color_quant v1.1.0
   Compiling strsim v0.10.0
   Compiling textwrap v0.15.0
   Compiling image v0.23.14
   Compiling digest v0.10.3
   Compiling ecdsa v0.13.4
   Compiling ethers-contract-derive v0.6.3
   Compiling phf v0.8.0
   Compiling colored v2.0.0
   Compiling clap v3.1.14
   Compiling dirs v2.0.2
   Compiling bytes v0.4.12
   Compiling sha-1 v0.8.2
   Compiling md-5 v0.9.1
   Compiling terminal_size v0.1.17
   Compiling glob v0.3.0
   Compiling unicode-width v0.1.9
   Compiling checked_int_cast v1.0.0
   Compiling console v0.15.0
   Compiling qrcode v0.12.0
   Compiling ws v0.9.2
   Compiling sha2 v0.10.2
   Compiling jsonrpc-core v18.0.0
   Compiling time v0.1.43
   Compiling similar v2.1.0
   Compiling chunked_transfer v1.4.0
   Compiling ureq v2.4.0
   Compiling chrono v0.4.19
   Compiling automerge v0.1.0 (https://github.com/automerge/automerge-rs?rev=291557a019acac283e54ea31a9fad81ed65736ab#291557a0)
   Compiling number_prefix v0.4.0
   Compiling lexopt v0.2.0
   Compiling indicatif v0.16.2
   Compiling dialoguer v0.10.1
   Compiling ansi_term v0.12.1
   Compiling colored_json v2.1.0
   Compiling linked-hash-map v0.5.4
   Compiling yaml-rust v0.4.5
   Compiling colored v1.9.3
   Compiling serde_yaml v0.8.24
   Compiling ed25519-zebra v3.0.0
   Compiling scrypt v0.8.0
   Compiling radicle-keystore v0.2.0
   Compiling automerge-backend v0.0.1 (https://github.com/automerge/automerge-rs.git?rev=e72571962b51c2f0726fb534890ef3b4f7c74dfc#e7257196)
   Compiling pbkdf2 v0.8.0
   Compiling coins-core v0.2.2
   Compiling scrypt v0.7.0
   Compiling k256 v0.10.4
   Compiling coins-bip32 v0.3.0
   Compiling automerge v0.0.2 (https://github.com/automerge/automerge-rs.git?rev=e72571962b51c2f0726fb534890ef3b4f7c74dfc#e7257196)
   Compiling coins-bip39 v0.3.0
   Compiling eth-keystore v0.3.0
   Compiling coins-bip32 v0.6.0
   Compiling ethers-providers v0.6.2
   Compiling ethers-etherscan v0.2.2
   Compiling ethers-solc v0.1.2
   Compiling walletconnect v0.1.0 (https://github.com/xphoniex/walletconnect-rs?branch=v0.1.0#82ab2922)
   Compiling ethers-contract v0.6.2
   Compiling coins-ledger v0.4.5
   Compiling ethers-signers v0.6.2
   Compiling ethers-middleware v0.6.2
   Compiling ethers v0.6.2
   Compiling git2 v0.13.25
   Compiling link-git v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling radicle-git-ext v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling radicle-macros v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling link-crypto v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling link-identities v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling link-replication v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling cob v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling link-tracking v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling librad v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling lnk-clib v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling radicle-git-helpers v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling lnk-profile v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling lnk-identities v0.1.0 (https://github.com/radicle-dev/radicle-link?rev=a9485b78b5c78d252c92f61d990cf34622d1c8f1#a9485b78)
   Compiling radicle-common v0.1.0 (/build/source/common)
   Compiling radicle-terminal v0.1.0 (/build/source/terminal)
   Compiling rad-sync v0.1.0 (/build/source/sync)
   Compiling rad-track v0.1.0 (/build/source/track)
   Compiling rad-init v0.1.0 (/build/source/init)
   Compiling rad-checkout v0.1.0 (/build/source/checkout)
   Compiling rad-untrack v0.1.0 (/build/source/untrack)
   Compiling rad-clone v0.1.0 (/build/source/clone)
   Compiling rad-rm v0.1.0 (/build/source/rm)
   Compiling rad-pull v0.1.0 (/build/source/pull)
   Compiling rad-push v0.1.0 (/build/source/push)
   Compiling rad-edit v0.1.0 (/build/source/edit)
   Compiling rad-ls v0.1.0 (/build/source/ls)
   Compiling rad-ens v0.1.0 (/build/source/ens)
   Compiling rad-auth v0.1.0 (/build/source/auth)
   Compiling rad-inspect v0.1.0 (/build/source/inspect)
   Compiling rad-remote v0.1.0 (/build/source/remote)
   Compiling rad-account v0.1.0 (/build/source/account)
   Compiling rad-self v0.1.0 (/build/source/self)
   Compiling rad-patch v0.1.0 (/build/source/patch)
   Compiling rad-help v0.1.0 (/build/source/help)
   Compiling rad v0.1.0 (/build/source/rad)
   Compiling rad-merge v0.1.0 (/build/source/merge)
   Compiling rad-path v0.1.0 (/build/source/path)
   Compiling rad-issue v0.1.0 (/build/source/issue)
   Compiling rad-gov v0.1.0 (/build/source/gov)
   Compiling rad-reward v0.1.0 (/build/source/reward)
   Compiling radicle-cli v0.6.1 (/build/source)
error[E0433]: failed to resolve: use of undeclared crate or module `ethers_core`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_core`
    |
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared crate or module `ethers_contract`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_contract`
    |
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0432]: unresolved import `ethers_contract`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_contract`
    |
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error: cannot find attribute `ethcall` in this scope
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^
    |
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared crate or module `ethers_providers`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_providers`
    |
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared crate or module `ethers_contract`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_contract`
    |
    = help: consider importing one of these items:
            crate::ethereum::Lazy
            ethers::prelude::Lazy
            futures::future::Lazy
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared crate or module `ethers_contract`
   --> src/bin/git-remote-rad.rs:183:5
    |
183 | /     abigen!(
184 | |         EnsTextResolver,
185 | |         "[function text(bytes32,string) external view returns (string)]",
186 | |     );
    | |_____^ use of undeclared crate or module `ethers_contract`
    |
    = help: consider importing one of these items:
            crate::ethereum::Contract
            crate::ethereum::abi::Contract
            crate::ethereum::artifacts::Contract
            ethers::abi::Contract
            ethers::prelude::Contract
            ethers::prelude::artifacts::Contract
    = note: this error originates in the macro `abigen` (in Nightly builds, run with -Z macro-backtrace for more info)

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `radicle-cli` due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
```

### Notify maintainers

@amesgen

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.2.9, NixOS, 23.05 (Stoat), 23.05.20230403.53dad94`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/applications/version-management/radicle-cli/default.nix
+++ b/pkgs/applications/version-management/radicle-cli/default.nix
@@ -38,6 +38,9 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  # Otherwise, there are errors due to the `abigen` macro from `ethers`.
+  auditable = false;
+
   nativeBuildInputs = [
     pkg-config
     cmake

